src/tests/ftest/util/general_utils.py

```diff
@@ -40,6 +41,75 @@ class DaosTestError(Exception):
     """DAOS API exception class."""
 
 
+def run_command(command, timeout=60, verbose=True, raise_exception=True,
```

Comment:
I'm just curious. Where do you use this method?

Reply:
Not using it in my test, I added it as part of another change so it can land when this ticket lands.